tests/series/test_size_bytes.py

```diff
@@ -226,4 +229,17 @@ def test_series_struct_size_bytes(size) -> None:
     s = Series.from_arrow(data)
 
     assert s.datatype() == DataType.from_arrow_type(dtype)
-    assert s.size_bytes() == data.nbytes + 8 + (size * 8) // 2 + math.ceil(size / 8)
+
+    # check validity bitmaps were were properly propagated to children
+    if size > 0:
+        child_arrays = [data.field(i) for i in range(data.type.num_fields)]
+        child_arrays_without_validity_buffer = len(
+            [child for child in child_arrays if any([b is None for b in child.buffers()])]
+        )
+        assert s.size_bytes() == (
+            data.get_total_buffer_size()
+            + child_arrays_without_validity_buffer * math.ceil(size / 8)
+            + additional_bytes_for_large_string_offsets
+        )
+    else:
+        assert s.size_bytes() == data.get_total_buffer_size() + additional_bytes_for_large_string_offsets
```

Comment:
Note that tests in this test file are cleaned up much better in a follow-on PR: https://github.com/Eventual-Inc/Daft/pull/865